pvlib/pvsystem.py

```diff
@@ -2537,6 +2537,48 @@ def pvwatts_losses(soiling=2, shading=3, snow=0, mismatch=2, wiring=2,
     return losses
 
 
+def combine_loss_factors(index, *losses, fill_method='ffill'):
+    r"""
+    Combines Series loss fractions while setting a common index.
+
+    The separate losses are compounded using the following equation:
+
+    .. math::
+
+        L_{total} = 1 - [ 1 - \Pi_i ( 1 - L_i ) ]
+
+    :math:L_{total} is the total loss returned
+    :math:L_i is each individual loss factor input
+
+    Note the losses must each be a series with a DatetimeIndex.
+    All losses will be resampled to match the index parameter using
+    the fill method specified (defaults to "fill forward").
+
+    Parameters
+    ----------
+    index : DatetimeIndex
+        The index of the returned loss factors
+
+    *losses : Series
+        Each parameter is a series of fractions to be compounded
+
+    fill_method : {None, ‘backfill’/’bfill’, ‘pad’/’ffill’, ‘nearest’}
```

Comment:
Are the opening/closing single quotes important here? Looks like a minor issue the pandas docstring is rendered, assuming that is cut/pasted here. I think single quote ' is ok for both, I could be wrong.

Reply:
+1 to not using "fancy" quotation marks.  Suggest also specifying the default value rather than listing None, e.g.:

`fill_method : {'backfill'/'bfill', 'pad'/'ffill', 'nearest'}, default 'backfill'`